index.js

```diff
@@ -90,18 +91,25 @@ const onConfig = (config, argv) => {
 
   // livereload settings
   if (config.livereload) {
-    on('serve', () => onLivereloadConfig(slidePipeline, config))
+    on('serve', () => {
+      openurl.open('http://localhost:' + config.port)
```

Comment:
because this is inside `if (config.livereload) {}` block, this only works when livereload is true. I prefer creating new option `open: boolean` for this feature.

Reply:
And when we add an option, we need to document that in README.md.

Reply:
Thanks for the feedback. I moved `openurl` out of the livereload block further down below all the other assets. It's hard for me to tell exactly where the correct place to open the browser is. I ran it a few times with different presentations and it seems to be in the right spot. I can't tell if I'm just getting lucky race conditions.

I updated the README to describe this runs with `-b` and `--openBrowser`. `-o` for open was already taken with `-o`.

Thank you!